src/tests/sc-test.c

```diff
@@ -85,16 +85,19 @@ int sc_test_init(int *argc, char *argv[])
 		} else {
 			for (i = rc = 0; rc != 1 && i < (int) sc_ctx_get_reader_count(ctx); i++)
```

Comment:
Should not the check be updated also here in `rc != 1`?

Reply:
Amended, thank you